randovania/games/prime2/logic_database/Agon Wastes.txt

```diff
@@ -1988,20 +1994,34 @@ Extra - in_dark_aether: True
                   Dark World Damage ≥ 20 and Activate Safe Zone
           Standable Terrain (Intermediate) and Dark World Damage ≥ 20 and Use Screw Attack (Space Jump)
   > Event - Dark Agon Temple Key Gate
-      # https://www.youtube.com/watch?v=Tn9a6XtRnB4&list=PLPFZVy8SRe3tAkQMzVw8vWt_w_AF2_l91&index=45
+      # https://www.youtube.com/watch?v=yd91qBUxCeA
       Dark Agon Key 1 and Dark Agon Key 2 and Dark Agon Key 3 and Morph Ball Bomb and Morph Ball and Space Jump Boots and Bomb Space Jump (Expert) and Single Room Out of Bounds (Expert) and Slope Jump (Expert) and Standable Terrain (Expert) and Dark World Damage ≥ 175
+  > Amorbis Trigger
+      All of the following:
+          Knowledge (Intermediate)
+          Any of the following:
+              Space Jump Boots or Bomb Space Jump (Intermediate) or Bomb Jump (Intermediate)
```

Comment:
Missing Morph and Bombs

Reply:
this is to reach the trigger, and then the bombs would be handled with the fight. I didn't think it was necessary to add them here. Want me to add them still?

Reply:
I guess that is true, but it feels wrong/inconsistent to do it this way.